### ISSUE TITLE
The genome_configs directory is now out of the package scope

### ISSUE THREAD

**Issue by standage** [BrendelGroup/AEGeAn#225]
I just made a fresh install of AEGeAn on my personal laptop, and I'm troubleshooting some install issues.

```
$ fidibus -h
Traceback (most recent call last):
  File "/Users/standage/anaconda3/envs/aegean/bin/fidibus", line 4, in <module>
    __import__('pkg_resources').run_script('LocusPocus==0+unknown', 'fidibus')
  File "/Users/standage/anaconda3/envs/aegean/lib/python3.7/site-packages/pkg_resources/__init__.py", line 667, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/Users/standage/anaconda3/envs/aegean/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1463, in run_script
    exec(code, namespace, namespace)
  File "/Users/standage/anaconda3/envs/aegean/lib/python3.7/site-packages/LocusPocus-0+unknown-py3.7.egg/EGG-INFO/scripts/fidibus", line 17, in <module>
    import LocusPocus
  File "/Users/standage/anaconda3/envs/aegean/lib/python3.7/site-packages/LocusPocus-0+unknown-py3.7.egg/LocusPocus/__init__.py", line 47, in <module>
    test_registry = registry.Registry()
  File "/Users/standage/anaconda3/envs/aegean/lib/python3.7/site-packages/LocusPocus-0+unknown-py3.7.egg/LocusPocus/registry.py", line 29, in __init__
    self.update(genome_configs, clear=True)
  File "/Users/standage/anaconda3/envs/aegean/lib/python3.7/site-packages/LocusPocus-0+unknown-py3.7.egg/LocusPocus/registry.py", line 45, in update
    raise FileNotFoundError(message)
FileNotFoundError: config directory "/Users/standage/anaconda3/envs/aegean/lib/python3.7/site-packages/LocusPocus-0+unknown-py3.7.egg/LocusPocus/../genome_configs" does not exist
```

This seems to be related to the recent migration of `./LocusPocus/LocusPocus/genomes/` to `./LocusPocus/genome_configs/`. This line was no doubt updated to compensate for that change.

https://github.com/BrendelGroup/AEGeAn/blob/fff8f0b5d2a410511687665257e776090b4e5731/LocusPocus/LocusPocus/registry.py#L28

The problem is that the `genome_configs/` directory is now out of the package scope—the `./LocusPocus/LocusPocus/` directory—and into (for lack of a better word) the "project" scope (the `./LocusPocus/` directory). As such, I wouldn't expect adding the `../` prefix to the path in the `resource_filename` command to work under normal conditions. When `python setup.py install` is run, it can only install code and package data from within the package scope (the `./LocusPocus/LocusPocus` directory). That's what this line was doing before.

https://github.com/BrendelGroup/AEGeAn/blob/fff8f0b5d2a410511687665257e776090b4e5731/LocusPocus/setup.py#L40

Now that `genome_configs/` is out of the package scope, there's no way for `setup.py` and company to recognize and install it along with the other package components. To be honest, I'm surprised it's working at all on your systems, and I've been unable to tell *how* you've made it work by looking at the updated install and Singularity build scripts.

In the short term, I'm curious how `fidibus` finds the `genome_configs/` directory in your current setup. But for the long term, my suggestion would be to return the directory to the package scope.

**Comment by vpbrendel** [comment]
Thanks - you are correct in your review, and I can confirm that package currently fails. I must have tested a different version ...
An additional complication is that singularity/squashfs is broken in the latest Fedora 30 kernels just installed on almost all of our systems.
I'll get that fixed this weekend.

**Comment by vpbrendel** [comment]
Actually, it seems like there is simple fix to the problem other than moving directories. I had explored this before, but somehow did not include it in the previous submission. The current pull request should close the issue.

**Comment by vpbrendel** [comment]
One caveat to the above: in registry.py, the Unit Tests may still fail. I am not sure how they work, but "LocusPocus/genome_configs" may not be the correct path. Easy enough for Daniel to check, adjust, and test ...

**Comment by standage** [comment]
Fixed by #226.

I'm closing this thread for now. There are indeed problems with the test suite(s), but these appear unrelated and of a larger scope. Tracking in #228.
